src/topology/bounded_continuous_function.lean

```diff
@@ -51,11 +51,11 @@ def mk_of_discrete [discrete_topology α] (f : α → β) (hf : ∃C, ∀x y, di
 
 /-- The uniform distance between two bounded continuous functions -/
 instance : has_dist (α →ᵇ β) :=
-⟨λf g, Inf {C | C ≥ 0 ∧ ∀ x : α, dist (f x) (g x) ≤ C}⟩
+⟨λf g, Inf {C | 0 ≤ C ∧ ∀ x : α, dist (f x) (g x) ≤ C}⟩
```

Comment:
This is the same as `⨆ x, dist (f x) (g x)`. @sgouezel, did you have any reason to use this form?

Reply:
I wondered this too!

Reply:
I want a definition that also works when alpha is empty.

Reply:
We have `Sup ∅ = 0`.

Reply:
I know, but this is a not obvious design choice, and some users might not know this (maybe I didn't know at the time I wrote this line!). Instead of relying on this, I think it is better to avoid all ambiguities with an explicit description like this one.